test/test_images/latencymako/sender_receiver.go

```diff
@@ -72,6 +74,24 @@ type paceSpec struct {
 	duration time.Duration
 }
 
+type attackSpec struct {
+	pacer    vegeta.Pacer
+	duration time.Duration
+}
+
+func (a attackSpec) Attack(i int, targeter vegeta.Targeter, attacker *vegeta.Attacker) {
+	printf("Starting attack %d° with pace %v rps for %v seconds", i+1, a.pacer, a.duration)
+	res := attacker.Attack(targeter, a.pacer, a.duration, fmt.Sprintf("%s-attack-%d", eventsSource(), i))
+	for _ = range res {
+	}
+
+	// Wait for flush
+	time.Sleep(1 * time.Second)
```

Comment:
As discussed, we should think of a more reliable way to ensure events are not being delivered, like using a select with a ticker and reset the ticker on every received event, but a sleep should be a good enough first step 👍